readme.md

```diff
@@ -41,20 +40,58 @@ Refer to [readme](Gems/ROS2ScriptIntegration/readme.md)
 
 # RobotecWatchdogTools
 
-Minimal dependency Gem that allows to setup runtime checks and prevents starting the Editor/GameLauncher if the requirements are not met
+Minimal dependency Gem that allows the setup runtime checks and prevents starting the Editor/GameLauncher if the requirements are not met
 Refer to [readme](Gems/RobotecWatchdogTools/readme.md)
 
+# Disable the main view
+
+Deprecated, please use `-console-mode` switch available in o3de 2409. 
+The description of said feature is in the original [PR](https://github.com/o3de/o3de/pull/18093).
 
 # LevelModificationTools
 
-TBD
+The level modification tool contains a component called PrefabVariantEditorComponent. 
+This component allows the change a variant of loaded prefab during game mode.
+It exposes PrefabVariantRequestsBus to Script Canvas or LUA.
 
 # SensorDebug
 
-A tool that allows to adjust frequency, activate and deactive sensor during game mode.
+A tool that allows the adjusted frequency, activate and deactivate sensor during the game mode.
 
 # Smoothing
 
 Gem contains a smoothing component that will mimic the movement of an attached entity with the tracked entity. It offers multiple smoothing methods. It allows the lock Z axis to point up direction.
-Useful for robots' movement smoothing.
-![alt text](doc/Smoothing.png) 
\ No newline at end of file
+Useful for robots' movement smoothing.\
+![alt text](doc/Smoothing.png) 
+
+# ExposeConsoleToRos
+
+Simple utility Gem that exposes the O3DE console to ROS 2 topics.
+It has two std_msgs/msg/String topics:
+```
+\o3de_console_out
+\o3de_console_in
+```
+Currently `o3de_console_in` is usable only.
+The gem functionality is available only in Profile/Debug.
+
+# Pointcloud
+
+A Gem that introduces point clouds to O3DE.
+It offers:
+- PointcloudFeatureProcessor with public API
+- Pointcloud product asset 
+- A public API for configuration (`PointcloudConfigurationBus`)
+
+At this moment it accepts [PLY](https://en.wikipedia.org/wiki/PLY_(file_format)) as a source asset.
+
+![](doc/Pointcloud.png)\
+Pointcloud asset was obtained from [potree](https://github.com/potree/potree).
+
+# RobotecSpecatorCamera
+
+A configurable spectator camera, an alternative to the Fly Camera component from O3DE.
```

Comment:
I believe that after rebase this part won't be needed  - RobotecSpectatorCamera has been added to README #67